packages/editor/src/components/collab-sidebar/utils.js

```diff
@@ -0,0 +1,9 @@
+/**
+ * Sanitizes a comment string by removing non-printable ASCII characters.
+ *
+ * @param {string} str - The comment string to sanitize.
+ * @return {string} - The sanitized comment string.
+ */
+export function sanitizeCommentString( str ) {
+	return str.trim().replace( /[^\x20-\x7E]/g, '' );
+}
```

Comment:
@poojabhimani12 Can you comment on why this is needed? Seems to be only used for a disabled state.

Reply:
We are using this to disable the comment button, so if a user adds only spaces without entering any text, the comment button will remain disabled until a valid input is provided.

Reply:
And why do we need to remove `[^\x20-\x7E]`?